designer/client/component.js

```diff
@@ -45,6 +46,14 @@ function ComponentField(props) {
   return <Base>{props.children}</Base>;
 }
 
+function NationalInsuranceField() {
+  return (
+    <ComponentField>
+      <div className="box" />
+    </ComponentField>
+  );
+}
+
 function TextField() {
```

Comment:
since these two components display the same thing, we can just remove `function NationalInsuranceField`, and in the componentTypes object, change `NationalInsuranceField: NationalInsuranceField` to `NationalInsuranceField: TextField`

Reply:
Resolved.